tests/src/test/scala/pathy/path.scala

```diff
@@ -100,133 +84,194 @@ class PathSpecs extends Specification with ScalaCheck {
     ).toList must_== List("c", "foo", "bar", "flat.md")
   }
 
-  "parseRelFile - image.png" in {
-    parseRelFile("image.png") must beSome(file("image.png"))
-  }
+  "parseRelFile" >> {
```

Comment:
If that's the case, I think you want some kind of `testAll` rather than a generator.

But I doubt that there are really too many tests that are really common. As soon as you have a `/` you work only for posixCodec, right?